sale_blanket_order/models/blanket_orders.py

```diff
@@ -659,9 +659,6 @@ def _compute_quantities(self):
     def _validate(self):
         try:
             for line in self:
-                assert (
```

Comment:
Ok, even if in an Odoo context, using derived from 'UserError' exceptions is better.